3rdparty/CMakeLists.txt

```diff
@@ -36,6 +36,9 @@ add_subdirectory(openssl openssl EXCLUDE_FROM_ALL)
 # volk
 if (Vulkan_FOUND)
 	set(VOLK_PULL_IN_VULKAN ON)
+	if (WIN32)
+		set(VOLK_STATIC_DEFINES VK_USE_PLATFORM_WIN32_KHR)
+	endif()
```

Comment:
is there other stuff for other platforms?

Reply:
I repeat my question

Reply:
nah